collectors/o365/o365_collector.js

```diff
@@ -73,29 +77,63 @@ class O365Collector extends PawsCollector {
     
```

Comment:
Could you please also remove `collectorId: "none",` from `regValues`

Reply:
actually you can remove all of these values because PAWS will use them for registration for you.
But we probably want to add Azure tenant id and Azure audit streams into reg values.